build/circle_tests.sh

```diff
@@ -3,4 +3,6 @@
 set -x
 set -e
 
+docker run -t --entrypoint="/usr/bin/run_unittests" -w "/root/src/preprocessing-workflow" oesteban/fmriprep
```

Comment:
Great suggestion! I won't act on it now because not trying to find a perfect solution here, especially because I'm supposed to be working on fieldmap stuff--I just want to close this pull request because long-running pr's are bad form in my book. It is true that the test infrastructure (e.g., files in the `build` directory) in this package is a little wonky. Maybe create an issue for un-wonkifying the test infrastructure?